### PR TITLE
Update dgpix.cpp

### PR DESCRIPTION
**Pull request by HeroponRikiBestest**
Notes undumped game called Beat Player 2000.

### CODE REVIEW

**Comment by DavidHaywood** [comment]
There's also "Let's Dance"
http://blog.system11.org/?p=1396

**Comment by HeroponRikiBestest** [comment]
Okay, so  fishing maniac runs the type a rev2 subboard and let's dance runs the bmkey Rev 2 subboard, and they both use the minus rev5 board. I'm guessing fishing maniac 2 is on the same hardware since 1 and 3 are; is it know what board and subboard it uses? Furthermore, are there any other undumped dgpix games I'm missing?

File: src/mame/drivers/dgpix.cpp [hunk 1]
```diff
@@ -15,6 +15,8 @@
  [*] the version number is written in the flash roms at the beginning of the game settings
 
 Note: There is known to exist an alternate version of The X-Files titled The Sex Files which is undumped
+An additional game called Beat Player 2000 is undumped, using the VRenderO Minus Rev5 PCB,
+with an undocumented BMkey Flash Rev2 dgPIX Entertainment 1999 subboard.
```
Comment (cuavas) [inline src/mame/drivers/dgpix.cpp:19]: It’s zero, not oh.  Also, use descriptive PR titles/descriptions and commit messages.  Given this very basic error, I’m not confident in the veracity of the comment.  Can you provide a citation for the information?

**Review by cuavas — CHANGES_REQUESTED** [review]

File: src/mame/drivers/dgpix.cpp (hunk 1 above)
Comment (HeroponRikiBestest) [reply src/mame/drivers/dgpix.cpp:19]: https://twitter.com/dan_evileye/status/723152981009399808?s=19 and http://blog.system11.org/?p=1396. I'm going to add let's dance (which david already linked), as well as Fishing Maniac http://blog.system11.org/?p=1396 and Fishing Maniac 2 http://blog.system11.org/?p=2182. I'm sorry about the bad PR name and commit message, I confess I've barely used github. I also apologize for the error.

Comment (angelosa) [reply src/mame/drivers/dgpix.cpp]: Notice the whole driver uses the _VRenderO_ cargo cult naming, and should be bulk renamed.

Comment (DavidHaywood) [reply src/mame/drivers/dgpix.cpp]: VRenderO Minus, it's important to specify

VRender0 = Crystal System, done in hardware
VRender0 Minus = this, done on a Hyperstone CPU in software

Comment (angelosa) [reply src/mame/drivers/dgpix.cpp]: fwiw gp2x also uses SoC solutions derivative of the VRender3D design

Comment (HeroponRikiBestest) [reply src/mame/drivers/dgpix.cpp]: > Notice the whole driver uses the _VRenderO_ cargo cult naming, and should be bulk renamed.

Should I do that here, too? Or leave someone else to do it.

Comment (HeroponRikiBestest) [reply src/mame/drivers/dgpix.cpp]: I replaced all instances of VRenderO with VRender0 in the driver. This PR should now be finalized.

**Comment by cuavas** [comment]
The PR title and commit messages are still not descriptive.

**Comment by simzy39** [comment]
I've just found a variant of X-Files being called 'adult' version: https://www.ebay.com/itm/223289867954?hash=item33fd1cfeb2:g:kxMAAOSwK5tcGUMf

It needs to be documented in the driver.

**Comment by DavidHaywood** [comment]
that just looks like the most common version of the game, the current parent set.

**Comment by cuavas** [comment]
@Osso13 please don’t merge PRs that have completely non-descriptive titles.  Someone needs to make the title meaningful before release time.

**Comment by Osso13** [comment]
Ah, so updating it while squashing and merging isn't enough. Ok, will avoid doing this in the future, point taken.

**Comment by cuavas** [comment]
Squashing and merging allows you to correctly format/clean up the commit message, but the PR still needs to have a meaningful title/description.  It’s a pretty basic thing for contributors to do, so let’s encourage it.

**Pull request merged**